src/components/elements/off-canvas-menu.tsx

```diff
@@ -167,573 +168,586 @@ export const OffCanvasMenuWrapper = (props: OffCanvasMenuWrapperProps) => {
-                    <AccordionItem
-                        header={<Localize translate_text="Trade" />}
-                        header_style={header_style}
-                        style={content_style}
-                    >
-                        {!props.is_ppc && (
-                            <>
-                                <Text color="grey-5" mb="8px" size="14px">
-                                    {<Localize translate_text="Trade types" />}
-                                </Text>
-                                <Flex mb="2rem">
-                                    <NavCard
-                                        aria_label="CFDs"
-                                        icon={() => (
-                                            <img
-                                                src={CFD}
-                                                alt="CFDs trade type"
-                                                width="32"
-                                                height="32"
-                                            />
-                                        )}
-                                        content={
-                                            <Localize translate_text="Trade with leverage and tight spreads for better returns on successful trades." />
-                                        }
-                                        title={<Localize translate_text="CFDs" />}
-                                        onClick={handleArrowClick}
-                                        to="/trade-types/cfds/"
-                                    />
-                                </Flex>
-                                {is_row && (
-                                    <Flex mb="2rem">
-                                        <NavCard
-                                            aria_label="Options"
-                                            icon={() => (
-                                                <img
-                                                    src={Options}
-                                                    alt="Options trade type"
-                                                    width="32"
-                                                    height="32"
-                                                />
-                                            )}
-                                            content={
-                                                <Localize translate_text="Earn a range of payouts without risking more than your initial stake." />
-                                            }
-                                            title={<Localize translate_text="Options" />}
-                                            onClick={handleArrowClick}
-                                            to="/trade-types/options/"
+    const accordion_data: Array<AccordionDataProps> = [
+        {
+            id: 'navbar_01',
+            title: '_t_Trade_t_',
+            component: (
```

Comment:
Over 300 lines of code is a lot for a React component. Since this component is 822 lines let's make it more simple and create sub-components in different files instead of declaring all of them in here.